tests/system/providers/google/cloud_sql/example_cloud_sql.py

```diff
@@ -272,64 +235,72 @@
     # ### INSTANCES TEAR DOWN ###################### #
     # ############################################## #
 
     # [START howto_operator_cloudsql_replicas_delete]
     sql_instance_failover_replica_delete_task = CloudSQLDeleteInstanceOperator(
-        project_id=GCP_PROJECT_ID,
         instance=FAILOVER_REPLICA_NAME,
         task_id='sql_instance_failover_replica_delete_task',
     )
 
     sql_instance_read_replica_delete_task = CloudSQLDeleteInstanceOperator(
-        project_id=GCP_PROJECT_ID, instance=READ_REPLICA_NAME, task_id='sql_instance_read_replica_delete_task'
+        instance=READ_REPLICA_NAME, task_id='sql_instance_read_replica_delete_task'
     )
     # [END howto_operator_cloudsql_replicas_delete]
+    sql_instance_failover_replica_delete_task.trigger_rule = TriggerRule.ALL_DONE
+    sql_instance_read_replica_delete_task.trigger_rule = TriggerRule.ALL_DONE
 
     # [START howto_operator_cloudsql_delete]
     sql_instance_delete_task = CloudSQLDeleteInstanceOperator(
-        project_id=GCP_PROJECT_ID, instance=INSTANCE_NAME, task_id='sql_instance_delete_task'
-    )
-    sql_instance_delete_task2 = CloudSQLDeleteInstanceOperator(
-        project_id=GCP_PROJECT_ID, instance=INSTANCE_NAME2, task_id='sql_instance_delete_task2'
+        instance=INSTANCE_NAME, task_id='sql_instance_delete_task'
     )
     # [END howto_operator_cloudsql_delete]
+    sql_instance_delete_task.trigger_rule = TriggerRule.ALL_DONE
+
+    delete_bucket = GCSDeleteBucketOperator(
+        task_id="delete_bucket", bucket_name=BUCKET_NAME, trigger_rule=TriggerRule.ALL_DONE
+    )
 
     (
-        sql_instance_create_task
-        >> sql_instance_create_2_task
+        # TEST SETUP
+        create_bucket
+        # TEST BODY
+        >> sql_instance_create_task
         >> sql_instance_read_replica_create
         >> sql_instance_patch_task
-        >> sql_instance_patch_task2
         >> sql_db_create_task
-        >> sql_db_create_task2
         >> sql_db_patch_task
-        >> sql_db_patch_task2
         >> sql_gcp_add_bucket_permission_task
         >> sql_export_task
-        >> sql_export_task2
         >> sql_gcp_add_object_permission_task
-        >> sql_gcp_add_bucket_permission_2_task
         >> sql_import_task
-        >> sql_import_task2
         >> sql_db_delete_task
-        >> sql_db_delete_task2
         >> sql_instance_failover_replica_delete_task
         >> sql_instance_read_replica_delete_task
         >> sql_instance_delete_task
-        >> sql_instance_delete_task2
+        # TEST TEARDOWN
+        >> delete_bucket
     )
 
     # Task dependencies created via `XComArgs`:
     #   sql_instance_create_task >> sql_gcp_add_bucket_permission_task
-    #   sql_instance_create_2_task >> sql_gcp_add_object_permission_task
-    #   sql_instance_create_2_task >> sql_gcp_add_bucket_permission_2_task
+    #   sql_instance_create_task >> sql_gcp_add_object_permission_task
+
+    from tests.system.utils.watcher import watcher
```

Comment:
I think it would be useful to add something like
````
### Everything below this line is not part of example ###
```
to make users people do not copy this code to their DAGs or do not ask questions "do I need it"? Let' remember that example dags are mostly examples, their use for system tests is secondary.